graphql-kotlin-schema-generator/build.gradle.kts

```diff
@@ -14,6 +14,8 @@ dependencies {
     implementation("io.github.classgraph:classgraph:$classGraphVersion")
     testImplementation("io.reactivex.rxjava3:rxjava:$rxjavaVersion")
     testImplementation("org.junit.jupiter:junit-jupiter-params:$junitVersion")
+    // TODO remove after graphql-java 16.1 release
+    api("com.google.guava:guava:30.0-jre")
```

Comment:
see https://github.com/graphql-java/graphql-java/issues/2107 for details

Reply:
16.1 has been released